src/components/Search.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import "../styles/Search.css";
+import { ReactComponent as SearchIcon } from "../assets/searchIcon.svg";
+import { getImages } from "../requests/getImages";
+
+export default function Search({ setSearchResults }) {
+  const [value, setValue] = useState("");
+
+  const changeValue = (event) => setValue(event.target.value);
+
+  const handleSearch = async () => {
+    const imageResult = await getImages(value);
+    return setSearchResults(imageResult);
+  };
+
+  return (
+    <>
+      <form action="" className="search--form">
```

Comment:
Theres no need for the empty `action` property here. What you could do is replace this with the `onSubmit` event and specify your existing handleSearch function. This means when you press enter and this field is in focus, the form is submitted and you can call that function!